### PR TITLE
Added transactionId and batchId arguments

### PR DESCRIPTION
**Pull request by sandisk**
And moved header render to a separate method

### CODE REVIEW

File: src/Feed.php [hunk 1]
```diff
@@ -25,13 +25,29 @@ class Feed implements RenderableInterface
...
+     *
+     * @var string
+     */
+    private $transactionId;
+
+    /**
      * Creates a new feed document, for the specified timestamp (defaults to now if empty)
      *
      * @param $timestamp
      */
```
Comment (gabriel403) [inline src/Feed.php]: update heredoc!

**Comment by gabriel403** [comment]
Other than comment :+1:

**Comment by sandisk** [comment]
@gabriel403: Updated

**Comment by gabriel403** [comment]
:+1:

**Pull request merged**